pages/SignUp.js

```diff
@@ -0,0 +1,18 @@
+import BasePage from '../pages/BasePage';
+
+class SignUp extends BasePage {
+
+  //WebElements
+  get headerSignUp() {return $('strong')}
```

Comment:
Same comment about `strong` being a tipically weak selector. Choose something that idenitifies this element more uniquely